### ISSUE TITLE
Stay online on focused Telegram window

### ISSUE THREAD

**Issue by kosmodrey** [telegramdesktop/tdesktop#2094]
When you stop typing or move mouse for a while (~30 seconds) Telegram changes your status to "offline", chat member sees your status as "recently seen", but you still have focused chat window and can receive and read all his messages. As you start typing or move mouse, your status returns back to "online".

Disable "inactive timer" on focused Telegram chat window. It's so annoying! Ghaah! :angry:

**Comment by auchri** [comment]
That's a feature. E.g. if you afk, you should not be online ;)

**Comment by kosmodrey** [comment]
30 seconds (!) is too small for afk. It would be nice to have an option to disable this or make this timeout longer. We don't have this feature on mobile clients (iOS/Android) while app is open.

**Comment by stek29** [comment]
related to #1518?

**Comment by kosmodrey** [comment]
@stek29 yes, but it's not the best solution. Maybe there's a way to specify user config file (cli argument maybe)?

**Comment by stek29** [comment]
No, but you can make a PR after adding it.

**Comment by erisdev** [comment]
oh good, does this mean telegram desktop has already solved overtake/telegram#279? i don't know why i haven't just switched to this client already.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 383 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>
